src/core/atomicdex/services/mm2/mm2.service.cpp

```diff
@@ -2095,8 +2134,8 @@ namespace atomic_dex
         if (it == m_balance_informations.cend())
         {
             ec = dextop_error::unknown_ticker;
-            SPDLOG_INFO("Invalid Ticker {}", ticker);
-            return "Invalid Ticker";
+            SPDLOG_INFO("Ticker {} is invalid or not yet enabled", ticker);
+            return "Ticker not yet enabled";
```

Comment:
is "Invalid Ticker" check here https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/Dex/Settings/RecoverSeedModal.qml#L375 related? If so, needs to be changed too.

Reply:
Checking that area now to suppress balance/history requests until zhtlc completely activated.